### PR TITLE
Removing submodule and dataScience

### PR DESCRIPTION
**Pull request by rha930**
- refactoring all dataScience -> gamechangerml
- removed submodule and dataScience entirely from this repo.

### CODE REVIEW

**Comment by vctrstrm** [comment]
> * refactoring all dataScience -> gamechangerml
> * removed submodule and dataScience entirely from this repo.

Can we do `gamechanger_ml` with underscore, for readability?

**Comment by vctrstrm** [comment]
There's a sister PR here for `gamechanger-ml` repo: https://github.com/dod-advana/gamechanger-ml/pull/25

## Additional changes:
* Changed README.md to match
* Fixed references to assets found in gamechangerml repo and generally removed things irrelevant to gamechanger-data repo
* Removed gamechangerml as submodule entirely, and all references in configs/gitignore/dockerignore
* Revised `paasJobs/configure_repo.sh` script to handle setting up and installing/initializing gamechangerml as a package in way that keep deployments simple (probably need to revisit this later for local installs - e.g. figuring out how to handle topic model dependencies with no dev/prod access)

## Testing

Successfully deployed repo as per instructions in repo-level readme on DEV data eng box and used document parser to covert 10 pdfs to jsons.

File: paasJobs/configure_repo.sh [hunk 1]
```diff
@@ -88,6 +102,7 @@ fi
 EOF
 
 install_app_config
+ensure_gamechangerml_is_installed
 install_topic_models
```
Comment (meglynkau) [inline paasJobs/configure_repo.sh:106]: Does the installation of gamechangerml require the topic model? If so installing topic models should come before occur before installing gamechangerml repo. @rha930 @vctrstrm

**Review by meglynkau — CHANGES_REQUESTED** [review]

File: paasJobs/configure_repo.sh (hunk 1 above)
Comment (vctrstrm) [reply paasJobs/configure_repo.sh:106]: nope, topic model is post-install

**Review by meglynkau — APPROVED** [review]

**Pull request merged**